src/core/aws-server-http_utils.adb

```diff
@@ -1310,6 +1307,7 @@ package body AWS.Server.HTTP_Utils is
                        Line_Attribute.Reference;
 
       Status_Code : Messages.Status_Code := Response.Status_Code (Answer);
+      With_Body   : constant Boolean := Messages.With_Body (Status_Code);
```

Comment:
I always put the constant before the variable.

Reply:
This constant here because uses just declared variable.

Reply:
I'll move it to Send_Data and style will be fixed

Reply:
Done